starlette_admin/fields.py

```diff
@@ -276,6 +276,68 @@ def additional_js_links(self, request: Request, action: RequestAction) -> List[s
         return []
 
 
+@dataclass
+class CKEditor4Field(TextAreaField):
```

Comment:
CKEditor 4 is no longer supported since June 2023. Continuing to use it may cause compatibility, security, and performance issues.

Reply:
As far as I know there are people using it and offering this one as an "option" lookgs good but you are right to be concerned. Isn't there any way to enlight people about the risk but at the same time offer this field?

Reply:
I think it's best to avoid the pain of supporting a library that has reached its end of life. Those who still want to use it can create a custom field

Reply:
Alright 🚀